projects/rehype-typst/lib/index.js

```diff
@@ -113,7 +113,7 @@ export default function rehypeTypst(options) {
         const defaultEm = 11;
         const height = parseFloat(root.children[0].properties['dataHeight']);
         const width = parseFloat(root.children[0].properties['dataWidth']);
-        const shift = height - result.baselinePosition;
+        const shift = height - result.baselinePosition + 2.8; /* line up baselines */
```

Comment:
this seems like a hack fwiw. is this documented anywhere?

Reply:
yeah this seems hacky, given that it depends on users' `line-height`, so this doesn't makes sense to me.

probably a better solution is to target the CSS class for the result rendered equation.